OpenRA.Mods.Common/Traits/Buildings/Exit.cs

```diff
@@ -46,16 +46,14 @@ public Exit(ActorInitializer init, ExitInfo info)
 
 	public static class ExitExts
 	{
-		public static Exit FirstExitOrDefault(this Actor actor, string productionType = null)
+		public static Exit FirstExitOrDefault(this Actor actor, string productionType = null, Func<Exit, bool> p = null, WPos? nearestTo = null)
```

Comment:
Should we rename this now to `NearestOrDefault`?

Reply:
Since this is invoked like so `myActor.FirstExitOrDefault(...)`, instead of `ExitExts.FirstExitOrDefault(myActor, ...)` I think we should keep `Exit` in the name so that it reads `myActor.NearestExitOrDefault(...)` instead of `myActor.NearestOrDefault(...)` which doesn't immediately tell the reader what is being selected.